### PR TITLE
Albums/220412

### PR DESCRIPTION
**Pull request by Yesterday17**

### CODE REVIEW

File: album/COCX-41738.toml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+artist = "玲音（茅原実里）、詩花（高橋李依）、亜夜（日高里菜）"
+
+[[discs.tracks]]
+title = "なんどでも笑おう"
+artist = "天海春香（中村繪里子）、如月千早（今井麻美）、星井美希（長谷川明子）、萩原雪歩（浅倉杏美）、高槻やよい（仁後真耶子）、秋月律子（若林直美）、三浦あずさ（たかはし智秋）、水瀬伊織（釘宮理恵）、菊地真（平田宏美）、双海亜美（下田麻美）、双海真美（下田麻美）、四条貴音（原由実）、我那覇響（沼倉愛美）、奧空心白（田中あいみ）、亜夜（日高里菜）"
+
+[[discs.tracks]]
+title = "KAWAII ウォーズ (オリジナル・カラオケ)"
+type = "instrumental"
+
```
Comment (tuzi3040) [inline album/COCX-41738.toml:48]:
```suggestion
[[discs.tracks]]
title = "KAWAII ウォーズ (オリジナル・カラオケ)"
type = "instrumental"
artist = "天海春香（中村繪里子）、水瀬伊織（釘宮理恵）、菊地真（平田宏美）、我那覇響（沼倉愛美）、安部菜々（三宅麻理恵）、双葉杏（五十嵐裕美）、春日未来（山崎はるか）、田中琴葉（種田梨沙）、大崎甘奈（黒木ほの香）、大崎甜花（前川涼子）、奧空心白（田中あいみ）、詩花（高橋李依）"

```

File: album/COCX-41738.toml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+[[discs.tracks]]
+title = "ダンス・ダンス・ダンス (オリジナル・カラオケ)"
+type = "instrumental"
+
```
Comment (tuzi3040) [inline album/COCX-41738.toml:52]:
```suggestion
[[discs.tracks]]
title = "ダンス・ダンス・ダンス (オリジナル・カラオケ)"
type = "instrumental"
artist = "如月千早（今井麻美）、秋月律子（若林直美）、三浦あずさ（たかはし智秋）、四条貴音（原由実）、神崎蘭子（内田真礼）、高垣楓（早見沙織）、最上静香（田所あずさ）、白石紬（南早紀）、白瀬咲耶（八巻アンナ）、杜野凛世（丸岡和佳奈）、奧空心白（田中あいみ）、玲音（茅原実里）"

```

File: album/LACM-24251.toml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+[[discs.tracks]]
+title = "虹の行方"
+
+[[discs.tracks]]
+title = "Daybreak Age"
+
+[[discs.tracks]]
+title = "虹の行方 (Off Vocal)"
+artist = "シャイニーカラーズ"
+artists.composer = "三好啓太"
```
Comment (tuzi3040) [inline album/LACM-24251.toml:22]:
```suggestion
title = "虹の行方 (Off Vocal)"
artists.composer = "三好啓太"
```

File: album/LACM-24251.toml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+artists.arranger = "三好啓太"
+artists.lyricist = "古屋真"
+type = "instrumental"
+
+[[discs.tracks]]
+title = "Daybreak Age (Off Vocal)"
+artist = "シャイニーカラーズ"
+artists.composer = "桑原佑介"
```
Comment (tuzi3040) [inline album/LACM-24251.toml:30]:
```suggestion
title = "Daybreak Age (Off Vocal)"
artists.composer = "桑原佑介"
```

File: album/SCDC-00233.toml [hunk 5]
```diff
@@ -0,0 +1,29 @@
+[album]
+album_id = "d22b69df-6819-4083-b255-3b43f80b6c85"
+title = "Ever17 ドラマCD「2035」"
+artist = "ドラマ"
```
Comment (tuzi3040) [inline album/SCDC-00233.toml:4]: Any possibility to have specific values?

**Review by tuzi3040 — CHANGES_REQUESTED** [review]

File: album/SCDC-00233.toml (hunk 5 above)
Comment (Yesterday17) [reply album/SCDC-00233.toml:4]: 自动聚集上去的，我忘了.jpg

**Review by tuzi3040 — APPROVED** [review]

**Pull request merged**